next/components/ui/BlogPostsFilter/BlogPostsFilter.tsx

```diff
@@ -0,0 +1,158 @@
+import {
+  Enum_Pagecategory_Color,
+  PageCategoryEntityFragment,
+  TagEntityFragment,
+} from '@backend/graphql'
+import Chip from '@components/forms/simple-components/Chip'
+import { getCategoryColorLocalStyle } from '@utils/colors'
+import { isDefined } from '@utils/isDefined'
+import { useLocale, useTranslations } from 'next-intl'
+import React, { useEffect, useState } from 'react'
+import { Label, Selection, TagGroup, TagList, Text } from 'react-aria-components'
+
+interface BlogPostsFilterProps {
```

Comment:
Plase change interface to type (check first section in our docs: https://bratislava.github.io/frontend/components)